Formula/rtmidi.rb

```diff
@@ -34,10 +33,11 @@ class Rtmidi < Formula
   end
 
   def install
+    ENV.cxx11
     system "./autogen.sh", "--no-configure" if build.head?
     system "./configure", *std_configure_args
-    system "make"
     system "make", "install"
+    doc.install %w[doc/release.txt doc/html doc/images] unless build.head?
```

Comment:
```suggestion
    doc.install %w[doc/release.txt doc/html doc/images] if build.stable?
```

Reply:
I've also made this change to the [`rtaudio` PR](https://github.com/Homebrew/homebrew-core/pull/89794).